OneCommunityGlobal/HGNRest
# Jiayu limit see all

# Description
<img width="712" alt="image" src="https://github.com/OneCommunityGlobal/HGNRest/assets/58479595/d14079dc-ada4-4f25-a988-7cc02afa734e">

## Related PRS:
- This back PR is based on PR #731 
- To test this backend PR you need to checkout [frontend PR 2412](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2412).

## Main changes explained:
<img width="642" alt="image" src="https://github.com/OneCommunityGlobal/HGNRest/assets/58479595/f961617c-5112-44c3-af08-cd38d2caed14">

## How to test:
1. check into current branch
2. do `npm install` and `npm run build && npm run start` to run this PR locally
3. Please follow the testing steps of [frontend PR 2412](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2412).

## Screenshots or videos of changes:
https://www.loom.com/share/80d08159a46f40c8ab9871d27d729044?sid=1537e57a-a323-4124-84e8-37773be5d1b9
